epub34/rs/index.html

```diff
@@ -1205,22 +1205,24 @@ <h3>Cascading Style Sheets (CSS)</h3>
 							[=EPUB content documents=].</p>
 					</li>
 					<li>
-						<p id="confreq-css-overrides">SHOULD NOT override the EPUB creator's style sheets, but SHOULD do
-							so in a way that preserves the Cascade when necessary: through a user agent style sheet, or
-							[[html]] [^html-global/style^] attributes.</p>
+						<p id="confreq-css-overrides">MAY override parts of the creator's style sheets (for, e.g., ergonomic or 
+							user interface reasons). If it does, it SHOULD do so in a way that preserves the cascade model
+							of CSS: through a user agent style sheet, or [[html]] [^html-global/style^] attributes.
+							Reading system developers should also publicly document the user agent style sheets and how they
```

Comment:
Only open-source RSes do this. This cannot be a requirement IMO, but it could be a Note.

Reply:
It's back to a non-normative lowercase "should" so it's akin to a note. (Relying on case is a bad idea even if it's technically allowed, as it's not terribly accessible and generally confusing to readers, so if there's a way to rephrase and keep the emphasis it would be better.)

Reply:
I change the sentence using "it is expected" to avoid misunderstanding around the "should" word...